docs/hardware/devices/seeed-studio/esp32-sx1262-kit/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+id: xiao-esp32s3-kit
+title: XIAO ESP32S3 & Wio-SX1262 Kit
+sidebar_label: XIAO ESP32S3 & Wio-SX1262 Kit
+sidebar_position: 2
+---
+
+A thumb-sized LoRa dev kit, supporting WiFi, BLE, and LoRa, it includes a built-in power management chip and can be extended via IIC, UART, and other GPIO interfaces, with compatibility for Arduino development.
+
+
+
+
+### Specifications
+
+- **MCU**
+  - XIAO ESP32S3 (WiFi & Bluetooth)
+- **LoRa Transceiver**
+  - Semtech SX1262
+- **Frequency options**
+  - 862-930 MHz
+- **Connectors**
+  - USB-C
+  - Antenna
+     - Dedicated 2.4 GHz metal spring antenna for WiFi/Bluetooth
+     - U.FL/IPEX antenna connector for LoRa
+
+### Features
+
+
+- **Thumb-sized Compact Design**: 21 x 18mm, adopting the classic form factor of XIAO.
+- **Grove Connectors**: 400+ Grove-compatible GPIOs for flexible expansion options.
+
+### Resources
+
+- Purchase Links:
+  - International
+    - [Seeed Studio Online Store](https://www.seeedstudio.com/Wio-SX1262-with-XIAO-ESP32S3-p-5982.html)
+  - Distributor
+    - [DigiKey](https://www.digikey.com/en/products/detail/seeed-technology-co-ltd/114993369/24814479)
```

Comment:
That links to the T1000-E, it should be https://www.digikey.com/en/products/detail/seeed-technology-co-ltd/102010611/25835813